scripts/testing/city_runner/cts.py

```diff
@@ -48,7 +48,7 @@ def analyze_process_output(self):
         fail_single_pattern = re.compile(b"^(FAILED .+\.|.+FAILED\.?)$")
         fail_pair_pattern = re.compile(b"^FAILED (\d+) of (\d+) tests\.$")
         skipped_pattern = re.compile(
-            b"(.*Skipping test\.+|skipping|SKIPPED.+)$")
+            b"(.*Skipping test\.+|skipping|SKIPPED.+|No tests ran)$")
         zerorun_pattern = re.compile(b"^Tests completed: 0$")
```

Comment:
This `zerorun_pattern` also sounds like a `No tests ran` situation. Do we know the difference?

Reply:
Ignore me - `zerorun_pattern` seems special cased to handle doubles, which we don't want to touch